pkg/kubelet/OWNERS

```diff
@@ -1,4 +1,45 @@
-assignees:
-  - dchen1107
-  - vishh
-  - yujuhong
+approvers:
```

Comment:
What's the relationship of this OWNERS file against OWNERS under subdirectory? Do we still need this?